File: parcial3/grupos.py
def isEven(document):
    document2 = str(document)
    if document2.isnumeric():
        n1 = document % 1000
    else:
        n1 = ord(document[-1]) + ord(document[-2]) + ord(document[-3])

    n2 = n1 // 100 + n1 // 10 + n1 % 10
    if n2 % 2 == 0:
        return "Eres par"
    return "Eres impar"


print(isEven("ABCDFE"))
print(isEven(1018231453))

File: parcial3/grupos.js
function isEven(document) {
    let document2 = document.toString();
    let n1;
    
    if (!/^\d+$/.test(document2)) {
        n1 = document2.charCodeAt(document2.length - 1) + 
             document2.charCodeAt(document2.length - 2) + 
             document2.charCodeAt(document2.length - 3);
    } else {
        n1 = document % 1000;
    }
    
    let n2 = Math.floor(n1 / 100) + Math.floor((n1 % 100) / 10) + (n1 % 10);
    
    return n2 % 2 === 0 ? 'Eres par': 'Eres impar';
}


console.log(isEven("ABCDFE"))
console.log(isEven(1018231453))